backends/tfhe-cuda-backend/cuda/include/integer/integer.h

```diff
@@ -50,6 +50,7 @@ typedef struct {
   uint32_t num_radix_blocks;
   uint32_t max_num_radix_blocks;
   uint32_t lwe_dimension;
+  uint32_t num_radix_ciphertexts;
```

Comment:
shouldn't we use a new class here ? something like `CudaRadixCiphertextVectorFFI` ?